portable/GCC/ARM_CM23_NTZ/non_secure/portasm.c

```diff
@@ -441,8 +436,7 @@ void vClearInterruptMask( __attribute__( ( unused ) ) uint32_t ulMask ) /* __att
             "   mov r11, r7                                     \n" /* r11 = r7. */
             "   msr psp, r0                                     \n" /* Remember the new top of stack for the task. */
             "   subs r0, r0, #40                                \n" /* Move to the starting of the saved context. */
-            "   ldmia r0!, {r2-r7}                              \n" /* Read from stack - r2 = PSPLIM, r3 = LR and r4-r7 restored. */
-            "   msr psplim, r2                                  \n" /* Restore the PSPLIM register value for the task. */
+            "   ldmia r0!, {r3-r7}                              \n" /* Read from stack - r3 = LR and r4-r7 restored. */
```

Comment:
In line 431, cause there is no psplim context any more, so "adds r0, r0, #24" should be "adds r0, r0, #20" . Same issue in line 438, 366.

Reply:
Please look at the latest changes. We decided to keep a slot for PSPLIM in the context to avoid complicating the code too much.